packages/webview_flutter/webview_flutter/test/webview_flutter_test.dart

```diff
@@ -1139,6 +1139,7 @@ void main() {
         const WebView(
           initialUrl: 'https://youtube.com',
           gestureNavigationEnabled: true,
+          allowsLinkPreview: false,
```

Comment:
can we also test the default value is `true`?

Reply:
Removed the bool from this test 'creation' and created a new group for `allowsLinkPreview` with default, disable, and can change tests.